### ISSUE TITLE
Group not filtered

### ISSUE THREAD

**Issue by Mariusthvdb** [Mariusthvdb/custom-attributes#37]
**Describe the bug**

**To Reproduce**

group entities seem not to be filtered, see below error in screenshot

```
  by_entity_id:

    sensor.cpu_speed:
      - brand

    group.media_players_device_trackers:
      - all

    group.hub_device_trackers:
      - all
```
**Expected behavior**

fall attributes fine, no dropdown attributes box
**Screenshots**

<img width="698" alt="Scherm­afbeelding 2023-12-15 om 15 59 56" src="https://github.com/Mariusthvdb/custom-attributes/assets/33354141/469866bf-2936-4fcc-ad57-4537cef2c5ed">

<img width="588" alt="Scherm­afbeelding 2023-12-15 om 15 56 25" src="https://github.com/Mariusthvdb/custom-attributes/assets/33354141/c6222f87-0fac-43b7-bca7-08554e8c79d3">

<img width="558" alt="Scherm­afbeelding 2023-12-15 om 15 56 32" src="https://github.com/Mariusthvdb/custom-attributes/assets/33354141/b1510e1e-efc2-465e-aaa3-16111852d09c">

**Desktop (please complete the following information):**
 - OS: 2023.12.3
 - Browser Mac Safari
 - Version Versie 17.1.2 (19616.2.9.11.12)

**Additional context**

legacy group yaml:
```
group:
  hub_device_trackers:
    name: Hub device trackers
    all: true
    entities: &trackers
      - device_tracker.ha_main
      - device_tracker.synology_diskstation
      - device_tracker.iungo
      - device_tracker.solaredge
      - device_tracker.ikea_tradfri
      - device_tracker.philips_hue_1
      - device_tracker.philips_hue_2
      - device_tracker.philips_hue_3
      - device_tracker.plugwise_smile
      - device_tracker.somfy_tahoma
```

Dom Path to attributes Dropdown:

<img width="820" alt="Scherm­afbeelding 2023-12-15 om 15 58 44" src="https://github.com/Mariusthvdb/custom-attributes/assets/33354141/f8ec5011-a708-4b2b-86ab-bb70a71b075e">

Note that a group without a dropdown box also shows the same error:

<img width="573" alt="Scherm­afbeelding 2023-12-15 om 16 11 57" src="https://github.com/Mariusthvdb/custom-attributes/assets/33354141/2db40713-8b6a-4654-b853-f259baf28a58">

<img width="696" alt="Scherm­afbeelding 2023-12-15 om 16 12 05" src="https://github.com/Mariusthvdb/custom-attributes/assets/33354141/3ffede0e-715b-461d-aba8-0f3f42bc0e27">

these group members dont have any attributes to show, so the box is not rendered. the error is ugly though

**Comment by elchininet** [comment]
>Dom Path to attributes Dropdown:

I see, those entities have a `more-info-default` inside before the attributes. Do you know if this happens only with groups entities or is this something that can happen with multiple entities? Just to know if it is possible to make a separate flow for `more-info-group`.

>Note that a group without a dropdown box also shows the same error:
>these group members dont have any attributes to show, so the box is not rendered. the error is ugly though

Is that error coming from the plugin?, we are not throwing any error, so I suspect that is cominf from another place. Just open the error and check who triggered it.

**Comment by Mariusthvdb** [comment]
not sure, it begins and ends with custom_attributes:

<img width="1451" alt="Scherm­afbeelding 2023-12-15 om 16 30 03" src="https://github.com/Mariusthvdb/custom-attributes/assets/33354141/632c5827-fc75-4b76-9b85-48b7ed8ce11a">

**Comment by elchininet** [comment]
It seems to be coming from somewhere else. Just uninstall the plugin and try to open the same dialog, if the error appear you know for sure that it is unrelated.

> not sure, it begins and ends with custom_attributes, and for now I only see it on these groups:
> 
> <img alt="Scherm­afbeelding 2023-12-15 om 16 30 03" width="1451" src="https://private-user-images.githubusercontent.com/33354141/290865281-632c5827-fc75-4b76-9b85-48b7ed8ce11a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDI2NTUxNDUsIm5iZiI6MTcwMjY1NDg0NSwicGF0aCI6Ii8zMzM1NDE0MS8yOTA4NjUyODEtNjMyYzU4MjctZmM3NS00Yjc2LTliODUtNDhiN2VkOGNlMTFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjE1VDE1NDA0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY0YzUzZTA4MDkwMjAzM2U5N2MzMjYxN2E5NTIxM2RmYjEzYzY5ZDc0ZWE5MTU2ZTVjMDdlNTk0YmVlOTA1M2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3E-Gnt6w47_63VDLHZniLOKjq2btjbPQuLOEbz_lHSo">

**Comment by Mariusthvdb** [comment]
yep, still there after unloading..... will file an issue in Core.
but hope to get the group filter working nonetheless ;-)

**Comment by elchininet** [comment]
>yep, still there after unloading..... will file an issue in Frontend

Did you try to see if all the entities in that group actually exist/are available?

**Comment by Mariusthvdb** [comment]
yes, and they all show in the more info....

**Comment by Mariusthvdb** [comment]
perfect!! all filtered, very nice indeed

please merge into Main !

**Comment by elchininet** [comment]
@Mariusthvdb,
By the way, it is time that you try to the release the first version, so if there is a bug in the relase process it can be fixed. As there is not any version released you can try this first time with [allow-same-version](https://docs.npmjs.com/cli/v8/commands/npm-version#allow-same-version) flag specifying the version `1.0.0`. I expect a huge release note, so you should edit it later to remove all the repetitive things and let only the most relevant.

**Comment by Mariusthvdb** [comment]
Right!

Normally I manually add a resource to the release but not sure how this automated system works yet.

What do I do?

**Comment by elchininet** [comment]
@Mariusthvdb,
The regular commands are in #28, but for the first time and to avoid having a 1.x.x version without a 1.0.0 version released, it would be better to run (in main branch and with the latest status):

```bash
npm version 1.0.0 --allow-same-version
```

**Comment by Mariusthvdb** [comment]
My man you must teach me...

Where do I enter these commands?

**Comment by elchininet** [comment]
@Mariusthvdb,

In your console in your local machine, in the root of the repository.

Before anything, make sure you have `node` and `yarn` in your system, if not it will not work.

Then, install the dependencies of the project if you have not done it yet:

```bash
yarn install
```

An then:

```bash

npm version 1.0.0 --allow-same-version
```

**Comment by Mariusthvdb** [comment]
Uhm, I only ever use the  safari browser on my Mac…
I can use terminal, but how to go to the ‘root of the repository’

Maybe do that later and you release 1.0.0?
Hartelijke groet,

Marijn ten Harmsen van der Beek
The Future is here, and it’s a bright one!

+31 (0)6 12313322

HarmsenvanBeek<http://www.harmsenvanbeek.nl/> | About.me<https://about.me/marijnthvdb> | LinkedIn<http://www.linkedin.com/in/marijntenharmsenvanderbeek> | Facebook<https://www.facebook.com/marijn.tenharmsenvanderbeek>

Op 16 dec 2023 om 14:29 heeft Iván Pereira ***@***.***> het volgende geschreven:

﻿

@Mariusthvdb<https://github.com/Mariusthvdb>,

In your console in your local machine, in the root of the repository.

Before anything, make sure you have node and yarn in your system, if not it will not work.

Then, install the dependencies of the project if you have not done it yet:

yarn install

An then:

npm version 1.0.0 --allow-same-version

**Comment by elchininet** [comment]
Hey, no worries, I can do it.
You can check how to do it later.
Let me check if it goes without any issues.
Regards

**Comment by elchininet** [comment]
The `main` branch is protected even for `git push` and `git push --tags`. Remove the protection for admins or contributors first. I think that this option might be enabled at the moment:

<img width="792" alt="image" src="https://github.com/Mariusthvdb/custom-attributes/assets/3728220/553ad766-75a4-4f27-8cef-b01ab739ae07">

**Comment by Mariusthvdb** [comment]
no it was not enabled... I enabled it now, and will disable again

**Comment by elchininet** [comment]
Weird, let me try again

**Comment by elchininet** [comment]
No luck:

<img width="859" alt="image" src="https://github.com/Mariusthvdb/custom-attributes/assets/3728220/2055706c-b932-4da4-bdec-2a44ef6a9b84">

**Comment by Mariusthvdb** [comment]
require a PR? but how?
can you reach me on Discord?

**Comment by elchininet** [comment]
No, push tags and push the branch cannot be done through a pull request.
Try to set up this option and add you and me there:

<img width="791" alt="image" src="https://github.com/Mariusthvdb/custom-attributes/assets/3728220/b8b521df-5914-4a36-94ba-46bc739a30df">

**Comment by Mariusthvdb** [comment]
link? is that in Actions?

**Comment by elchininet** [comment]
No, it is not that option either it seems

**Comment by elchininet** [comment]
Try to check soem of these soltuions, we tried already some of them:
https://stackoverflow.com/questions/52589285/cannot-push-on-github-suddenly

**Comment by elchininet** [comment]
Some people recommend to remove the protection and create it again. This is the whole config of one of my repos, try to mimic it:

![image](https://github.com/Mariusthvdb/custom-attributes/assets/3728220/7f5594d6-f9eb-49f9-a235-1825ba043d29)

**Comment by Mariusthvdb** [comment]
I have set that now yes.

Must go will be back later

**Comment by Mariusthvdb** [comment]
done

**Comment by elchininet** [comment]
@Mariusthvdb,

No luck, I think that for now it is better to do manual releases until you get how to run the commands in your console.

<img width="868" alt="image" src="https://github.com/Mariusthvdb/custom-attributes/assets/3728220/4abaf5ef-003b-42eb-adea-ef48c03a9ba5">

**Comment by Mariusthvdb** [comment]
how odd,  copied your config exactly..

**Comment by Mariusthvdb** [comment]
@elchininet please try once again? its all open now

**Comment by elchininet** [comment]
>how odd, copied your config exactly..

Yep, but it seems that it is different when you are the owner than with a contributor. I‘ll send you the instructions to install Node and yarn in your machine, after that everything should be easier for you.

>@elchininet please try once again? its all open now

Let me check

**Comment by elchininet** [comment]
Yes, [it worked](https://github.com/Mariusthvdb/custom-attributes/actions/runs/7239614874/job/19721684163)

You can enable the protection again.

**Comment by elchininet** [comment]
But it failed, let me check what happened.

**Comment by elchininet** [comment]
Ah, yes, something else.

Go to settings > actions > general

And check this:

<img width="792" alt="image" src="https://github.com/Mariusthvdb/custom-attributes/assets/3728220/47afaf0c-65be-4d85-a00d-684914c80542">

**Comment by Mariusthvdb** [comment]
set

**Comment by elchininet** [comment]
Let me restart the workflow.

**Comment by elchininet** [comment]
Nice, [everything is working](https://github.com/Mariusthvdb/custom-attributes/actions/runs/7239614874/job/19721807786). The release [is already published](https://github.com/Mariusthvdb/custom-attributes/releases).
